### DISCUSSION TITLE
How to set Whitelist networks ?

### DISCUSSION THREAD

**Discussion by larrychenlawrence** [Chia-Network/chia-blockchain#6666] [Q&A]
1.0.6 Change Loge:
You can now whitelist networks and hosts so that a node can always connect with chosen peers.
but How to set Whitelist networks ?

**Comment by loppefaaret** [comment]
under the "full_node:" section you need the follow entry (please note 2 space indent):
```
  exempt_peer_networks:
  - 10.1.1.0
```
